diff -u -r fscrypt-0.3.3.orig/debian/control fscrypt-0.3.3/debian/control
--- fscrypt-0.3.3.orig/debian/control	2021-10-31 17:08:31.000000000 +0000
+++ fscrypt-0.3.3/debian/control	2022-05-25 17:10:13.551207700 +0000
@@ -5,7 +5,7 @@
 Uploaders:
  Paride Legovini <paride@debian.org>,
 Build-Depends:
- debhelper-compat (= 13),
+ debhelper-compat (= 12),
  dh-exec,
  dh-golang (>= 1.35),
  golang-any (>= 2:1.10~),
